atom/common/node_bindings.h

```diff
@@ -85,6 +85,9 @@ class NodeBindings {
   // Whether the libuv loop has ended.
   bool embed_closed_;
 
+  // Loop used when constructed in WORKER mode
+  uv_loop_t worker_loop_;
```

Comment:
It looks like this thing is only used to check if the loop is a worker loop.
But we can make the same check using `browser_env_  == WORKER`.
So would the code become simpler if it is removed?

Reply:
I think you might be looking at an older diff.

`worker_loop_` *is* the loop when the constructor gets passed a `Worker` mode

Reply:
> `worker_loop_` _is_ the loop when the constructor gets passed a `Worker` mode

Yes, but `uv_loop_ = &worker_loop_;`, so... 
I mean using two things instead of one seems too complex to me. I might be wrong though )

Reply:
The upstream reason for deprecating uv_loop_new() was to remove unnecessary heap allocs, so I followed their lead by aggregating a uv_loop_t.

But a single variable is clearer though and I'm fine with either approach. If you want, I'll remove `worker_loop_` and new/delete a uv_loop_t iff `WORKER` mode.

Reply:
> If you want

It's not a good reason ) Approved.